Sources/SWBUtil/ProcessInfo.swift

```diff
@@ -200,6 +200,15 @@ extension ImageFormat {
             return false
         }
     }
+
+    public var requiresSwiftModulewrap: Bool {
+        switch self {
+        case .macho:
+            return false
+        default:
```

Comment:
nit: can we prefer to list the cases explicitly? I like to stay away from `default` since it forces re-evaluation of use sites when new cases are added. Not that adding new formats is particularly likely, but I could at least see a new one for wasm being added in the future.

Reply:
This is really an is-not-macho check, I don't expect we will ever move away from module wrap for any future object formats